tests/shared/ContextIntegrationTestDefs.js

```diff
@@ -24,59 +24,69 @@ fluid.registerNamespace("gpii.tests.contextIntegration");
 fluid.require("%gpii-universal");
 gpii.loadTestingSupport();
 
-
-fluid.defaults("gpii.tests.contextIntegration.environmentChangedRequestType", {
-    gradeNames: "kettle.test.request.http",
-    path: "/environmentChanged",
-    method: "PUT"
-});
-
 // NOTE: This inherits from from "gpii.test.testCaseHolder" via "gpii.test.integration.testCaseHolder.linux"
 // from which it gets loginRequest, logoutRequest and other standard events
 fluid.defaults("gpii.tests.contextIntegration.testCaseHolder", {
```

Comment:
With the removal of ContextManager, this file can be removed too.

Reply:
> Removed ContextManager files should be removed from [.nycrc](https://github.com/klown/universal/blob/GPII-3117/.nycrc#L9-L11).

Okay, done.
> 
> Searching the code base with `activeContext` shows 12 matches in a few files. Please consider to change them to `activePrefsSet`.

I could only find three matches for "activeContext" (singular), but maybe that's because I had already removed files or sections of files with "conditions" or "ContextManger", etc.  The three remaining matches were all relative to the `MatchMaker`.  Here are the links, where `activeContexts` has been replaced with `activePrefsSets`:

- [matchmaker.output.payload.json](https://github.com/klown/universal/blob/GPII-3117/gpii/node_modules/flowManager/test/data/matchmaker.output.payload.json#L2)
- [MatchMakerUtilitiesTests.js](https://github.com/klown/universal/blob/GPII-3117/gpii/node_modules/matchMakerFramework/test/js/MatchMakerUtilitiesTests.js#L345)
- [MatchMakerUtilities.js](https://github.com/klown/universal/blob/GPII-3117/gpii/node_modules/matchMakerFramework/src/MatchMakerUtilities.js#L50)

I'm still pondering that last one because, so far, I can't see where it's used.

Reply:
@cindyli I think I've found a bug, and I looking close to confirm.  Here's an summary of the "problem".  Note that the links below are to the current master branch.

The MatchMaker utility `gpii.matchMakerFramework.utils.preProcess()` allocates an  [`activeContexts[]`](https://github.com/GPII/universal/blob/master/gpii/node_modules/matchMakerFramework/src/MatchMakerUtilities.js#L50) array.

Later the cloud-based FlowManager's SettingsGetHandler performs the match in `gpii.flowManager.cloudBased.matchToSettings()`.  The first operation there is to filter the `finalPayload` in terms of keys such as `gpiiKey`, `preferences`, and others.  In particular, it looks for the `activeContextNames` key.  Note the difference between `activeContexts` and `activeContextNames`.  I don't think the SettingsGetHandler ever retrieves the `activeContexts`.

However, I've put in some log messages, and am running all of the tests to see what `matchToSettings()` actually does. I'll let you know the results.